deploy/topic-klage-vedtak-fattet-dev.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: vedtak-fattet
+  namespace: klage
+  labels:
+    team: klage
+  annotations:
+    dcat.data.nav.no/title: "Vedtak fattet på klage"
+    dcat.data.nav.no/description: "Fullførte vedtak på klageinstans som er klare for førsteinstans postes på dette topicet."
+spec:
+  pool: nav-dev
+  config:  # optional; all fields are optional too; defaults shown
+    cleanupPolicy: delete  # delete, compact
+    minimumInSyncReplicas: 1
+    partitions: 1
+    replication: 3  # see min/max requirements
+    retentionBytes: -1  # -1 means unlimited
```

Comment:
Nå ble jeg litt usikker, men gir det mening å angi både retentionBytes og retentionHours? Er det i såfall race to the top eller race to the bottom?

Reply:
Jeg slo opp i dokumentasjonen, og det vil fungere slik at hvis man har satt begge deler, så vil den slette når en av dem inntreffer, det er en OR ikke en AND. Det burde jeg kanskje klart å gjette meg til.. :)